progress/angular/components/PUBLISHING.md

```diff
@@ -0,0 +1,13 @@
+# Publishing Instructions
+
+To update the version number, edit the version in `package.json`. 
+
+To publish a new package through NPM, run the following commands from the root folder: 
+
+```
+yarn build
+cd ../dist
```

Comment:
is the dist folder a directory above where you run the build command?

Reply:
Yep, the dist folder is generated 1 folder above where the build command is ran.  I'm not sure why, but I was just looking at how things currently work.